grails-app/assets/javascripts/sites.js

```diff
@@ -470,6 +470,15 @@ var PidLocation = function (l, options) {
 
         }
     }
+
+    function getOldLayers (knownShapeConfig) {
+        var result  = {};
+        (knownShapeConfig || []).forEach(function(layer) {
+            layer.previousLayers && $.extend(result, layer.previousLayers);
```

Comment:
I've been trying to prefer underscore to jquery for utility functions with the general trend of libraries removing jquery dependencies